src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -175,7 +179,29 @@ int aliceVision_main(int argc, char** argv)
         }
     }
     std::vector<std::shared_ptr<sfmData::View>> targetViews;
-    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+
+    if (!byPass)
```

Comment:
Later, we access targetViews assuming it always has the same size as groupedViews without checking that it could be empty.
```
        for(std::size_t g = 0; g < groupedViews.size(); ++g)
        {
            std::shared_ptr<sfmData::View> hdrView = std::make_shared<sfmData::View>(*targetViews[g]); // <<<<
```